src/StackExchange.Redis/RedisServer.cs

```diff
@@ -819,6 +831,18 @@ public Task<EndPoint[]> SentinelGetSentinelAddressesAsync(string serviceName, Co
             return ExecuteAsync(msg, ResultProcessor.SentinelAddressesEndPoints);
         }
 
+        public EndPoint[] SentinelGetSlaveAddresses(string serviceName, CommandFlags flags = CommandFlags.None)
```

Comment:
Is this the analog to `SentinelGetMasterAddressByName`? The naming seems off here, I'd expect `ByName` in here (honestly, we'd re-think all of these API additions as a whole and consistently if we had a time machine)

Reply:
No, it's analogous to the method right above it that I copied it from `SentinelMasters`. In redis it translates to `SENTINEL slaves mymaster` and `SENTINEL sentinels mymaster`. While `SentinelGetMasterAddressByName` translates to `SENTINEL get-master-addr-by-name mymaster`

Reply:
On the API surface this is very confusing, it _looks_ like it's 1:1 with the underneath, since this is public API we should get it right. @mgravell thoughts?